bin/serv.js

```diff
@@ -93,6 +95,7 @@ if (args['ssl-cert'] && args['ssl-key']) {
 
 // get deault port from environment or else fallback
 args.port = Number.parseInt(args.port || (process.env.PORT || 5000), 10);
+args.host = args.host || "0.0.0.0";
```

Comment:
Use single quotes here for text, and if possible above in options too

Reply:
Done